includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -8673,6 +8547,7 @@ class AMP_Allowed_Tags_Generated {
 							'allow_relative' => true,
 							'protocol' => array(
 								'data',
+								'http',
```

Comment:
See https://github.com/ampproject/amphtml/pull/21686 and https://github.com/ampproject/amp-wp/pull/2051.